src/Sulu/Bundle/ContentBundle/Resources/views/Template/macros/add_button.html.twig

```diff
@@ -2,5 +2,7 @@
     <div id="mapper-add-{{ property.name }}" class="addButton" data-mapper-add="{{ property.name }}">
         <span class="control-label">Add {{ property.name }}</span>
     </div>
-    <span class="desc">(<span id="current-counter-{{ property.name }}">{{ property.minOccurs }}</span> out of <span>{{ property.maxOccurs }}</span>)</span>
+    {% if property.maxOccurs is not null %}
+    <span class="desc">(<span id="current-counter-{{ property.name }}">{{ property.minOccurs|default(1) }}</span> out of <span>{{ property.maxOccurs }}</span>)</span>
```

Comment:
The same is true for the other span 3 lines before... The problem is that I can't use the translator in here, because the locale is set to a wrong value. Would have a look at that in a separate PR.